packages/component-library/components/Paragraph/Paragraph.elm

```diff
@@ -0,0 +1,225 @@
+module Paragraph.Paragraph exposing
+    ( Config
+    , DataAttribute
+    , TypeVariant(..)
+    , a
+    , addDataAttribute
+    , classNameAndIHaveSpokenToDST
+    , div
+    , h1
+    , h2
+    , h3
+    , h4
+    , h5
+    , h6
+    , id
+    , p
+    , pre
+    , span
+    , tag
+    , variant
+    , view
+    )
+
+import CssModules exposing (css)
+import Debug exposing (log)
+import Html exposing (text)
+import Html.Attributes
+import List
+
+
+type Config msg
+    = Config (ConfigValue msg)
+
+
+type alias Element msg =
+    List (Html.Attribute msg) -> List (Html.Html msg) -> Html.Html msg
+
+
+type alias DataAttribute =
+    { name : String
+    , value : String
+    }
+
+
+type alias ConfigValue msg =
+    { tag : Maybe (Element msg)
+    , variant : TypeVariant
+    , id : Maybe String
+    , dataAttribute : List DataAttribute
+    , classNameAndIHaveSpokenToDST : Maybe String
+    }
+
+
+defaultConfig : ConfigValue msg
+defaultConfig =
+    { tag = Nothing
+    , variant = Body
+    , id = Nothing
+    , dataAttribute = []
+    , classNameAndIHaveSpokenToDST = Nothing
+    }
+
+
```

Comment:
We'd probably want to share this code eventually in some kind of helper file, but let's wait for the third occurrance in rule of three style.

Reply:
Heavy plus. Let's consider this for the next cycle